docs/src/apireference.md

```diff
@@ -169,6 +170,15 @@ The value of the attribute is of type `TerminationStatusCode`.
 TerminationStatusCode
 ```
 
+### Conflict Status
+
+The `ConflictStatus` attribute indicates why the optimizer stopped executing.
+The value of the attribute is of type `ConflictStatusCode`.
+
+```@docs
+ConflictStatusCode
```

Comment:
I would add `compute_conflict!` here and move all attributes to this section

Reply:
Better this way?

Reply:
👋